shared/src/dto/mockExamQuestionAttempt/mockExamQuestionAttempt.dto.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNumber, IsOptional } from 'class-validator';
+
+export class MockExamQuestionAttemptDto {
+  @ApiProperty()
+  @IsNumber()
+  questionId: number;
+
+  @ApiProperty()
```

Comment:
Why don't we help out swagger with determining the value of this field? If you use a union (ie. number | undefined) it doesn't exactly know to do and defaults to null (which isn't a valid value either way). If you add @ApiProperty({ type: 'number' }) it'll show it properly in the swagger playground